tools/samm-cli/src/main/java/org/eclipse/esmf/AspectSearchCommand.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package org.eclipse.esmf;
+
+import java.io.File;
+import java.util.List;
+
+import org.eclipse.esmf.aspectmodel.AspectModelFile;
+import org.eclipse.esmf.aspectmodel.loader.AspectModelLoader;
+import org.eclipse.esmf.aspectmodel.resolver.FileSystemStrategy;
+import org.eclipse.esmf.aspectmodel.resolver.ResolutionStrategy;
+import org.eclipse.esmf.aspectmodel.resolver.fs.FlatModelsRoot;
+import org.eclipse.esmf.aspectmodel.scanner.AspectModelScanner;
+import org.eclipse.esmf.aspectmodel.scanner.FileSystemScanner;
+import org.eclipse.esmf.aspectmodel.urn.AspectModelUrn;
+import org.eclipse.esmf.metamodel.AspectModel;
+import org.eclipse.esmf.metamodel.ModelElement;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+@CommandLine.Command( name = AspectSearchCommand.COMMAND_NAME,
+      description = "Search separate Properties files used by Aspect Model",
```

Comment:
I don't think we search Properties files (*.properties). Maybe: "Shows where model elements are used in Aspect Models" or similar?
Maybe "search" is also not optimal as a command name, IMO it is not really a search function, maybe we can call it "usage". Or something else?

Reply:
Yes, here you are totally right, 100% we have to change it for better explanation.

I think, that 'usage' as command name and 'Shows where model elements are used in Aspect Models' like description would be great!